### PR TITLE
Fix: Reset items opacity when copying other items

### LINKED ISSUE #12249: Bug: Cut items stay translucent after copying other items
### Description

Cut items stay translucent after copying other items, whereas they are expected to lose the translucency.
Below are two screen recordings, one is of Files that demonstrates the issue, and the other is of Windows' File Explorer that demonstrates the expected behavior.

https://user-images.githubusercontent.com/33163707/235434266-0553db91-f788-4fe8-b011-f32b1dd53b5a.mp4

https://user-images.githubusercontent.com/33163707/235434273-d7c14a11-8ca2-4ac4-8e8c-c2962e0bf7da.mp4

### Steps To Reproduce

Demonstrated in the recordings

### Requirements

-

### Files Version

2.4.70.0

### Windows Version

10.0.22621.1635

### Log file

[debug.log](https://github.com/files-community/Files/files/11364598/debug.log)

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12249 

I'm wondering if we should implement `Esc to cancel cut` as in file explorer....

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. Cut some items
   3. Select other items and select them
   4. See that items opacity resets

### CODE REVIEW

File: src/Files.App/Helpers/UIFilesystemHelpers.cs [hunk 1]
```diff
@@ -144,16 +131,19 @@ public static async Task CutItem(IShellPage associatedInstance)
...
+			var dataPackage = new DataPackage()
 			{
 				RequestedOperation = DataPackageOperation.Copy
 			};
-			ConcurrentBag<IStorageItem> items = new ConcurrentBag<IStorageItem>();
+			ConcurrentBag<IStorageItem> items = new();
 
-			if (associatedInstance.SlimContentPage.IsItemSelected)
+			if (associatedInstance.SlimContentPage.IsItemSelected &&
+				associatedInstance.SlimContentPage.SelectedItems is not null)
```
Comment (hishitetsu) [inline src/Files.App/Helpers/UIFilesystemHelpers.cs:141]: Why did you add this condition? I think it's redundant.
If this is correct, then `CopyItem()` should also be modified.

Comment (ferrariofilippo) [reply src/Files.App/Helpers/UIFilesystemHelpers.cs:141]: To resolve the null warnings. I can use ! if you think so

Comment (hishitetsu) [reply src/Files.App/Helpers/UIFilesystemHelpers.cs:141]: OK, I understand. Then we should do the same refactoring for `CopyItem()`.

Comment (hishitetsu) [reply src/Files.App/Helpers/UIFilesystemHelpers.cs:141]: Oh, I made a mistake. For `CutItem()`.

**Comment by ferrariofilippo** [comment]
> I'm wondering if we should implement `Esc to cancel cut` as in file explorer....

@hishitetsu what do you think about this?

**Comment by hishitetsu** [comment]
> > I'm wondering if we should implement `Esc to cancel cut` as in file explorer....
> 
> @hishitetsu what do you think about this?

I didn't know about this feature. I don't need it. I think it is unnecessary until someone requests it.

**Review by hishitetsu — APPROVED** [review]
LGTM
